scripts/install-chef-suse.sh

```diff
@@ -911,7 +911,7 @@ fi
 
 # Use current time zone
 (
-    . /etc/sysconfig/clock || : Ignoring missing /etc/sysconfig/clock
+    TIMEZONE=$(LC_ALL=C timedatectl status | grep "zone:" | cut -d: -f2 | cut -d' ' -f2)
```

Comment:
I don't really care about how it is implemented in this particular case either.  You could choose to see it as bikeshedding, or you could choose to see it as sharing knowledge in the hope that it's useful and can help produce better coding practices in the future.  Personally I find the latter type of collaboration more positive and enjoyable.  I mentioned this approach because `awk` seems generally underused in these kinds of situations, even though I find it can often get the job done more cleanly than standard combinations of `grep`, `cut`, `sed` etc., so I think it deserves some visibility.  If you already knew that then great.  I'm not trying to ram it down anyone's throat though.